src/TimeService/timeservice-rtems.h

```diff
@@ -0,0 +1,49 @@
+#include "interfaces_info.h"
+
+#define TICKS_PER_SECOND 1000000
+#define SCALER_VALUE 5
+
+void init_timer(Timer_Apbctrl1 *const timer, bool isChained, Timer_Id timerId)
+{
+   Timer_Apbctrl1_stop(timer);
+   Timer_Config config;
+   config.isEnabled = true;
+   config.isAutoReloaded = true;
+   config.isInterruptEnabled = false;
+   config.isChained = isChained;
+   config.reloadValue = UINT32_MAX;
+   Timer_Apbctrl1_init(timerId, timer, defaultInterruptHandler);
+   Timer_Apbctrl1_setBaseScalerReloadValue(timer, SCALER_VALUE);
+   Timer_Apbctrl1_setConfigRegisters(timer, &config);
+   Timer_Apbctrl1_start(timer);
+}
+
+static uint64_t swap_bytes(uint64_t x)
+{
+  uint64_t result;
+  result = ((x & 0x00000000000000FFULL) << 56) |
```

Comment:
At least small comment what is happening here would be nice

Reply:
It's just swapping endianess

Reply:
I added a comment